TRANSLATORS.md

```diff
@@ -0,0 +1,182 @@
+# TRANSLATORS.md
+
+Welcome! This guide is for anyone helping translate this project into other languages.
+
+Whether you're translating UI text or contributing localized content, this document explains how and where to make your changes.
+
+---
+
+## How Translations Work
+
+We use **Django's built-in internationalization (i18n)** framework to handle translations in both Python and JavaScript files.
+
+All translatable strings are extracted into `.po` files and compiled into `.mo` files, which Django uses at runtime.
+
+---
+
+## Translation File Locations
+
+- **Python & Templates:**
+  Translations are stored in `locale/<language_code>/LC_MESSAGES/django.po`.
+
+- **JavaScript files:**
+  Translations go into `locale/<language_code>/LC_MESSAGES/djangojs.po`.
+
+  Example:
+  ```
+  locale/
+    hi/
+      LC_MESSAGES/
+        django.po       # Python/template strings
+        djangojs.po     # JavaScript strings
+  ```
+
+---
+
+## Extracting Strings
+
+To extract translatable strings after making changes:
+
+```bash
+# For Python and templates
+python manage.py makemessages -l hi
+
+# For JavaScript
+python manage.py makemessages -d djangojs -l hi
+
+# Extract for all configured languages at once
+python manage.py makemessages -a
+
+# Extract JavaScript strings for all languages
+python manage.py makemessages -d djangojs -a
+```
+
+Replace `hi` with the target language code (e.g., `fr`, `es`, etc.).
+
+**Useful flags:**
+- `-a, --all`: Extract strings for all configured languages
+- `-d, --domain`: Specify domain (default: `django`, use `djangojs` for JavaScript)
+- `-l, --locale`: Specify a particular language code
+- `--help`: See all available options and flags
+
+For a complete list of options, run:
+```bash
+python manage.py makemessages --help
+```
+
+---
+
+## Compiling Translations
+
+After editing `.po` files, compile them so Django can use the updated translations:
+
+```bash
+python manage.py compilemessages
+```
+
```

Comment:
People probably don't need to run this manually, but I like having this here just in case they do.

```suggestion
After editing `.po` files, compile them so Django can use the updated translations:

```bash
python manage.py compilemessages
```

This is done automatically for you in development when you run the application, but you can also run it manually like this if needed.
```